specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,91 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST be a real number represented as an integer or a decimal value as defined in this section.
+
+## Attribute ID
+
+NumericFormat
+
+## Attribute name
+
+Numeric Format
+
+## Description
+
+Rules and formatting requirements for numeric columns appearing in billing data.
+
+## Requirements
+
+* Columns with a Numeric format MUST contain a single, numeric value.
+* Numeric values MUST be expressed as integer or decimal values. Fractional notation is NOT allowed.
+* Numerical values expressed with mathematical symbols, operators, or exponent values are not permitted.
+* Numeric values MUST NOT contain qualifiers or additional characters (e.g. currency symbols, units of measure, etc.).
+* Numeric values MUST NOT contain commas or punctuation marks except for a single decimal point if required to express a decimal value.
+* Numeric values MUST NOT include a character to represent sign for a positive value.  The only sign permitted is a negative sign (-) to indicate a negative value.
+* Numeric values MUST NOT be used to represent binary values (e.g. 0 or 1).
+
+## Numeric Precision
+
+The FOCUS specification does not require a specific level of precision for numeric values.  The level of precision required for a given column is determined by the provider and SHOULD be documented as a part of a data definition published by the provider.  The following table outlines the options for specifying the level of precision for a given metric in a data definition document.
```

Comment:
If we have to use BCP-14 normative content, it would be best if we can limit it to the requirements section so everything is in one place.

Reply:
Ok, the precision section is an interesting case because it's normative that providers SHOULD publish this, but I still think it's not an enforceable requirement.   I rearranged everything that is a MUST up top and left this at the end.   WDYT?